packages/application-components/src/components/page-content-containers/page-content-wide/page-content-wide.spec.tsx

```diff
@@ -0,0 +1,99 @@
+import { ThemeProvider } from '@commercetools-uikit/design-system';
+import { ReactNode } from 'react';
+import { screen, renderComponent } from '../../../test-utils';
+import PageContentWide, { TPageContentWide } from './page-content-wide';
```

Comment:
nit:
```suggestion
import type { ReactNode } from 'react';
import { screen, renderComponent } from '../../../test-utils';
import PageContentWide, { type TPageContentWide } from './page-content-wide';
```
(also applies to a few other places)

Reply:
Updated in this commit: 54eb045e0ab1c864e016dcbc99121f2623d17501